src/components/PokeCard.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+export default function PokeCard(props) {
+  return (
+    <div className="pokeCard">
+      <h1>{props.pokemonData.name}</h1>
+      <div className="pokemonImages">
+        <img src={props.pokemonData.sprites.front_default} alt="Default" />
+        <img src={props.pokemonData.sprites.front_shiny} alt="Shiny" />
+      </div>
+      <div className="pokemonControls">
+        <a href={props.pokemonData.video} target="blank">
```

Comment:
Following what I said before, it makes the code cleaner here using less navigation through the pokemon obj.
```suggestion
       PokeCard({ name, sprites, video })
        <a href={video} target="_blank" rel="noreferrer">
```
I know it is not much visible with this case but you'll see it for sure in more complex components.